space-cli/cmd/modules/operations/setup.go

```diff
@@ -285,3 +297,34 @@ func Setup(id, username, key, config, version, secret string, dev bool, portHTTP
 
 	return nil
 }
+
+func getContainerName(containerName string, id string) string {
```

Comment:
this should not take containerName as param.

Reply:
I made a common func for gateway and runner. it appends clusterid if its not default